java/core/src/java/org/apache/orc/InMemoryKeystore.java

```diff
@@ -330,4 +330,3 @@ private byte[] getMaterial() {
   }
 
 }
-
```

Comment:
Because these files have two newlines at the end, one newline is automatically deleted after spotless apply.

Reply:
> Because these files have two newlines at the end, one newline is automatically deleted after spotless apply.

Please make a new PR for the end line change separately before all of these migration.

Reply:
Also, we need a way to prevent any future regression in terms of the end line.

Reply:
> Please make a new PR for the end line change separately before all of these migration.

OK

> we need a way to prevent any future regression in terms of the end line

Now `spotless-maven-plugin` will check this behavior, so the beginning of the CI failed at the beginning.

https://github.com/apache/orc/actions/runs/8437582330/job/23107719304#step:5:8408